### PR TITLE
Several fixes on openchangeclient to send mails to OpenChange

### PR DESCRIPTION
**Pull request by sixstone-qq**
* Send `PidTagClientSubmitTime` while sending mail
* Send filename as unicode
* Send only available bytes while sending an attachment whose length is < 1024B

### CODE REVIEW

File: utils/openchangeclient.c [hunk 1]
```diff
@@ -895,6 +900,13 @@ static enum MAPISTATUS openchangeclient_sendmail(TALLOC_CTX *mem_ctx,
 		}
 	}
 
+	/* Send the submit time */
+	unix_to_nt_time(&nt, time(NULL));
+	now.dwLowDateTime = (nt << 32) >> 32;
+	now.dwHighDateTime = (nt >> 32);
+	set_SPropValue_proptag(&props[prop_count - 1], PR_CLIENT_SUBMIT_TIME, (void *)&now);
```
Comment (jkerihuel) [inline utils/openchangeclient.c]: Any specific reason why you chose not to use `set_SPropValue_proptag_date_timeval` from libmapi/property.c?

File: CHANGES.md [hunk 2]
```diff
@@ -13,6 +13,7 @@ Unreleased changes refer to our current [master branch](https://github.com/openc
 * Send event invitation mails to several attendees, mixing internal and external recipients
 * Fix folder hierarchy synchronization issues on mailbox subfolders
 * Old mails are now synchronized after account cleanup
+* Better support to attachments in OpenChange client against OpenChange server
```
Comment (jkerihuel) [inline CHANGES.md]: Can you please improve the CHANGELOG entry? I don't know what has been improved with this line only.

File: utils/openchangeclient.c [hunk 3]
```diff
@@ -715,6 +715,9 @@ static bool openchangeclient_stream(TALLOC_CTX *mem_ctx, mapi_object_t obj_paren
 	/* WriteStream operation */
 	printf("We are about to write %u bytes in the stream\n", bin.cb);
 	size = MAX_READ_SIZE;
+	if (size > bin.cb) {
+		size = bin.cb;
+	}
```
Comment (jkerihuel) [inline utils/openchangeclient.c]: :+1:

File: utils/openchangeclient.c (hunk 1 above)
Comment (sixstone-qq) [inline utils/openchangeclient.c]: I didn't know its existence. Thanks!

**Pull request merged**